aws/eks-claim.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: aws.platformref.upbound.io/v1alpha1
```

Comment:
@asarenkansah this is your environment specific right?

Reply:
Yes it is, going to change it here soon